packages/botonic-plugin-flow-builder/package.json

```diff
@@ -29,6 +29,7 @@
     "README.md"
   ],
   "devDependencies": {
+    "@types/axios": "^0.14.0",
```

Comment:
`axios` is on its 1.X.X version, should we update it?

Reply:
You are right, I don't know why but the other day I got an error that I didn't have the arios types. But now I have checked it and in the version we are using it is not necessary to import the @types separately.

Reply:
I will remove this dependency